data-hibernate-jpa/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
         exclude group:'io.micronaut.configuration', module:'micronaut-hibernate-jpa-spring'
         exclude group: 'io.micronaut.data'
     }
+    implementation "io.reactivex.rxjava2:rxjava:$rxJava2Version"
```

Comment:
why is `data-hibernate-jpa` pulling in rxjava?

Reply:
Looks like it was being used for tests